src/modules/flow/json/json.c

```diff
@@ -992,6 +993,7 @@ json_path_add_new_element(struct sol_flow_node *node, struct json_element *base_
             SOL_INT_CHECK(r, == -ENOMEM, r);
             SOL_INT_CHECK_GOTO(r, < 0, error_parse);
 
+            break;
```

Comment:
Maybe you should try to ping original authors of these codes - maybe they can remember some peculiarity. That said, I guess that _not_ having breaks is correct for json patch: note that both handle 'YXZ_BLOB' then falls through 'XYZ' case: it appears to prepare data for next case.